packages/ember-views/lib/views/text_field.js

```diff
@@ -54,6 +54,8 @@ export default Component.extend(TextSupport, {
     'width'
   ],
 
+  defaultLayout: null,
```

Comment:
This does make sense.  `<input>` cannot have child elements, this is just enforcing that (and preventing errors in IE8).